docs/user/attestation-provider.md

```diff
@@ -0,0 +1,113 @@
+
+??? note "Rewards"
+
+    Currently, the process of rewarding public attestation providers is still being discussed.
+
+## Prerequisites
+
+Before proceeding with the [installation](#installation), it is essential to follow every prerequisite.
+
+### System Requirements
+
+The following specifications were observed to be able to run all nodes on a single machine.
+Ideally, a single node should be run on each system in production for better performance and uptime.
+
+* **OS**: Ubuntu 20.04
+* **CPU Cores**: 16
+* **RAM**: 96GB
+* **Disk space**: 3TB with an option or plan in place to expand capacity. SSD is recommended because some chains like xrpl can fall out of sync on regular disks.
+
+As of Q2 2022, this is roughly what you can expect from each node regarding disk usage:
+
+| Volume              | Size  |
+|---------------------|-------|
+| algorand-data       | 1TB   |
+| algorand-indexer-db | 1TB   |
+| bitcoin-data        | 500GB |
+| dogecoin-data       | 60GB  |
+| litecoin-data       | 90GB  |
+| ripple-data         | 230GB |
+
+We are looking into ways to run a non-full history algorand node.
+
+### Dependencies
+
+* [`git`](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+* [`docker`](https://docs.docker.com/get-docker/)
+* [`docker-compose`](https://docs.docker.com/compose/install/)
+* [`openssl`](https://www.openssl.org/source/)
+* [`python3`](https://www.python.org/downloads/)
+
+### Node Setup
+
+The first step is to set up nodes for each supported network, that the attestation provider suite can then use to make attestations.
+
+The supported chains at the moment are the following:
+
+* [Bitcoin](https://github.com/bitcoin/bitcoin)
+* [Litecoin](https://github.com/litecoin-project/litecoin)
+* [Dogecoin](https://github.com/dogecoin/dogecoin)
+* [Rippled](https://github.com/ripple/rippled)
+* [Algorand](https://github.com/algorand/go-algorand)
+
+!!! warning "Setting the nodes up is a very long process"
+
+    Since the attestation provider suite requires having access to all supported networks, it is required to have an up-to-date node setup for each network.
+    Certain networks take very long times to get synchronized with, such as algorand which requires indexing the whole history of the blockchain locally.
+    In our testing it took a few hours to sync up with litecoin, dogecoin and xrpl, more than a day for bitcoin and weeks for algorand.
+
+#### Chain Configuration
+
+For this step, it is required to create a password that is at least 64 characters long.
+It is referred to later as `<your-password>`.
+
+##### Ubuntu
+
+The provided `install.sh` script was written for Ubuntu 20.04, so if you are on another platform follow the [alternate set of instructions](#other-platforms) or adapt the script to your platform.
```

Comment:
```suggestion
The provided `install.sh` script was written for Ubuntu 20.04, so if you are on another platform follow the [instructions below](#other-platforms) or adapt the script to your platform.
```

So it is clear we're jumping to another section and not another page.

Reply:
Fixed in [`5ce7d11` (#14)](https://github.com/flare-foundation/docs/pull/14/commits/5ce7d113176ef29848184f8caef20e8f801abaae)